src/core_ocean/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -596,6 +623,18 @@ subroutine ocn_vmix_coefs_cvmix_build(meshPool, statePool, forcingPool, diagnost
      endif !kpp stage 1 -- boundary layer compute
 
      if (kpp_stage == 2) then
+            ! update Langmuir enhancement factor
+            ! TODO: theory-wave is the only option to get Langmuir number and enhancement factor for now <06-09-18, Qing Li> !
+            if (config_cvmix_kpp_use_theory_wave .and. iceFraction(iCell) .lt. 0.05_RKIND) then
+               langmuirEnhancementFactor =  &
+                       cvmix_kpp_EFactor_model(windSpeed10m(iCell), &
+                                               surfaceFrictionVelocity(iCell), &
+                                               boundaryLayerDepth(iCell), &
+                                               cvmix_global_params)
+            else
+               langmuirEnhancementFactor = 1.0_RKIND
+            end if
+
```

Comment:
please move this down after L651, you may miss some boundary layer changes in your computation

Reply:
@vanroekel Done.